### PR TITLE
feat(new-design): allow the new SEKOIA.IO design

### PR DESCRIPTION
**Pull request by jadeen**
🤩

### CODE REVIEW

File: plugins/__pycache__/__init__.cpython-310.pyc
Comment (otetard) [inline plugins/__pycache__/__init__.cpython-310.pyc:1]: Could you remove these `.pyc` files please?

File: plugins/__pycache__/intakes_by_uuid.cpython-310.pyc
Comment (otetard) [inline plugins/__pycache__/intakes_by_uuid.cpython-310.pyc:1]: Same here.

File: plugins/__pycache__/redoc.cpython-310.pyc
Comment (otetard) [inline plugins/__pycache__/redoc.cpython-310.pyc:1]: Same here.

File: docs/assets/fonts/Inter-Black
[the remark this answers is not in the document]
Comment (jadeen) [reply docs/assets/fonts/Inter-Black:1]: I can't, it's the only way to load the font

[the remark this answers is not in the document]
Comment (jadeen) [reply docs/assets/fonts/Inter-Black:1]: Okay, I find my issue

**Review by otetard — APPROVED** [review]

**Pull request merged**